scripts/deploy.py

```diff
@@ -6,6 +6,7 @@
 
 def main():
     contract_name = str(os.getenv('CONTRACT')).lower()
+    chain_name = str(os.getenv('CHAIN')).upper()
```

Comment:
Would like to switch to chain IDs instead. We do the same for web app.
Chane names are inconsistent and error prone.

Consider the format `VOTER_${chain_id}=0x...0001` etc..